docs/HowTo/ManageKeys/AccountPlugins.md

```diff
@@ -4,6 +4,12 @@
 
 We recommended reading the [Plugins overview](../../Concepts/Plugins/Plugins.md) first to learn how to use plugins.
 
+## Available account plugins
+
+| Name | Version |  | Description |
+| --- | --- | --- | --- |
+| <span style="white-space:nowrap">`hashicorp-vault`</span> | `0.0.1` | <span style="white-space:nowrap">[Docs & Source](https://www.github.com/ConsenSys/quorum-account-plugin-hashicorp-vault)</span> | Enables storage of Quorum account keys in a Hashicorp Vault kv v2 engine.  Written in Go.
```

Comment:
Remove HTML please, see https://github.com/PegaSysEng/doc.common/wiki/MkDocs-And-Custom-Markdown-Guide#formatting-markdown-for-doc-site

Reply:
Thanks for the review.

Using HTML is supported in mkdocs.  In my opinion, HTML can improve the readability of published docs and/or source code in certain cases.

Additionally, as far as I'm aware, preventing the use of trailing whitespaces as well as HTML means that line breaks are not currently possible.

Can I ask the reasoning behind preventing the use of HTML?  Is it in case the docs are ever moved to a markdown engine that doesn't support HTML?